### DISCUSSION TITLE
I need Tips for OnComplete function

### DISCUSSION THREAD

**Discussion by Francaisedolon** [vydimitrov/react-countdown-circle-timer#250] [Q&A] [answered]
Hello,
I use the circle timer for a short time.
Its a cool tool. But I'm a newcomer on react etc.

I search a long time for ideas or demo about this.
But I don't found a solution.

How can I execute a function on the CircleTimer OnComplete event?
- ... make a div invisible
- ... make a page reload
- or any functions in Javascript?

Many thanks for your tips

Best Regards

**Comment by vydimitrov** [comment, marked as answer]
You can do this: 
```jsx
<CountdownCircleTimer
    isPlaying
    duration={7}
    onComplete={() => {
        // You can call your JS function here
    }}
  >
    {({ remainingTime }) => remainingTime}
  </CountdownCircleTimer>
```

**Reply by Francaisedolon** [reply]
Ok thanks
